packages/material/src/layouts/MaterialArrayLayout.tsx

```diff
@@ -257,6 +310,34 @@ export const mapDispatchToExpandPanelProps: (dispatch: Dispatch<AnyAction>) => D
         return array;
       })
     );
+  },
+  moveUp: (path: string, toMove: number) => (event: any): void => {
```

Comment:
here we could ideally reuse the move method from above to have less code

Reply:
From the above means? Are you referring to the helper method? or methods I created in packages/core/src/util/renderer.ts

Reply:
I meant the helper method from `packages/core/src/util/array.ts`